### ISSUE TITLE
ReadMe

### ISSUE THREAD

**Issue by sandysaders** [tj/channel.js#2]
Hey, this looks interesting. Could you add some examples to it? Also see the same for `wait group`and `deferred`. Handling some stuff at work where this actually seems like it could help a lot.

**Comment by tj** [comment]
yep, most have ./example files or dirs but I'll add some usage info to the readmes
